data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg

```diff
@@ -91,7 +93,7 @@
             caution=0.0
             grouping=no
         [/ai]
-        {INCOME 5 7 9}
+        {INCOME 3 6 9}
```

Comment:
Can you make this and the other one 4 6 8?

Reply:
ok, done in 3f99f17